src/main/send_main.cc

```diff
@@ -4,14 +4,90 @@
 
 #include "src/IPC/IPC.h"
 
-int main()
+#include "ipc_common.h"
+
+int main(int argc, char* argv[])
 {
-    char path[] = {"/home/maksim/ipc-class/input.txt"};
-    std::fstream file;
-
-    std::cout << "App: Launch Pipe Tx" << std::endl;
-    CreatorIPC* pipe_tx = new CreatorPipeTx();
-    file = pipe_tx->openReadFile(path);
-    SenderIPC* pipe_file_tx = pipe_tx->createIpcTx(&file);
-    std::cout << pipe_file_tx->send();
+    int c;
+    Arguments arg;
+
+    /* Default arguments values */
+    arg.tr_type = DEFAULT;
+    arg.read_path = NULL;
+    arg.write_path = NULL;
+
+    /* if not enough arguments given */
+    if (argc < 2) {
+        std::cerr << "ERROR: send_main requires arguments \n" << std::endl;
+        return -1;
+    }
+
+    while (1) {
```

Comment:
Maybe is worth to move this piece of code to separate file to avoid code duplication with receive_main.cc